src/python/pants/backend/helm/test/unittest/target_types.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.engine.target import COMMON_TARGET_FIELDS, Dependencies, MultipleSourcesField, Target
+
+
+class HelmUnitTestDependenciesField(Dependencies):
+    pass
+
+
+# -----------------------------------------------------------------------------------------------
+# `helm_unittests` target generator
```

Comment:
agh, this is in fact a misleading comment. This was initially implemented as a target generator (one per file) but then we went for a normal target because running the `unittest` plugin per test file was not very efficient. I'll remove the comment